src/isomorphic/React.js

```diff
@@ -62,6 +63,10 @@ var React = {
     only: onlyChild,
   },
 
+  CSS: {
+    multi: CSSPropertyOperations.multi,
+  },
```

Comment:
This would definitely be a ReactDOM API (nothing in `isomorphic/` should require anything in `renderers/`)